packages/jest-runner/testResources/exampleProject/src/Add.js

```diff
@@ -1,30 +1,21 @@
 // @ts-nocheck
-var __global_69fa48 = function (g) {
-  g.__mutantCoverage__ = g.__mutantCoverage__ || {
-    static: {},
-    perTest: {}
-  };
-
+var __global_69fa48 = (function(g){
+  if (g.__activeMutant__ === undefined && g.process && g.process.env && g.process.env.__STRYKER_ACTIVE_MUTANT__) {
+    g.__activeMutant__ = Number(g.process.env.__STRYKER_ACTIVE_MUTANT__);
```

Comment:
This was the issue. As you can see, the `process.env` variable name was out-dated here (`__ACTIVE_MUTANT__` -> `__STRYKER_ACTIVE_MUTANT__`). 

The code here was instrumented with an older version of the instrumenter.